1.0.0 changes

cs-abhishek-shukla opened the pull request:
Fixed typo, grammatical, and formatting issues

cs-suyog-jinturkar commented on README.md:
```diff
@@ -7,7 +7,7 @@
 # Overview
 
-This solution pack helps community users to submit their custom solution to FSR team.
+The **Submit Your Content** solution pack empowers community members to seamlessly submit their solutions for review on FortiSOAR’s Content Hub. By launching their FortiSOAR instance and leveraging playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions.
```
This section is very important and current details are not sufficient. This is an entry point to start using this solution. Please elaborate more on usecases, benefits in details, adding a widget screenshot, widget link, etc.

cs-abhishek-shukla replied:
Added more details

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -3,14 +3,15 @@
 
-By default SMTP is selected as a mode to send email. Can be changed through playbook, send email.
-<img src="./res/configuration.png" alt="Access Control" style="border: 1px solid #A9A9A9; border-radius: 4px; padding: 10px; display: block; margin-left: auto; margin-right: auto; width: 80%">
+The SMTP connector is configured to send email, by default. You can change your preference, on the connector to use for sending emails, by editing the playbook **Send Email**.
```
Please also mention PB collection and steps to be modified

cs-suyog-jinturkar commented on docs/usage.md:
```diff
@@ -1,5 +1,9 @@
 
+# Usage
+
+For detailed usage refer to the Submit Content Form widget.
```
Widget document link will be useful here

cs-yash-patodkar commented on README.md:
```diff
@@ -7,7 +7,7 @@
 # Overview
 
-This solution pack helps community users to submit their custom solution to FSR team.
+The **Submit Your Content** solution pack empowers community members to seamlessly submit their solutions for review on FortiSOAR’s Content Hub. By launching their FortiSOAR instance and leveraging playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions.
```
_By launching their FortiSOAR instance and leveraging playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions._

I dont think this is required can be modified to something like users can upload their cusstom solutions like connector widgets and sp's

cs-suyog-jinturkar commented on README.md:
```diff
@@ -7,7 +7,13 @@
+
+This solution pack simplifies content submission by breaking down the entire process into manageable steps addressed through user-friendly interfaces.
+
+- **Integrated with FortiSOAR**: Content submission is seamlessly integrated with FortiSOAR. You can use the [FortiSOAR RDK](https://docs.fortinet.com/fortisoar/connectors/rapiddevkit) to create solution packs, widgets, or connectors directly from the FortiSOAR interface. Once you've created your content, use the **Submit Your Content** solution pack to submit it to the FortiSOAR Integrations tea ...
```
to create solution packs, widgets, or connectors directly from the FortiSOAR interface ---> we don't create widget and sp using rdk. Also, rdk is not used from the FSR interface.

cs-abhishek-shukla committed:
1.0.0 changes
- Added usage section
- Rewrote Overview
- Removed other grammatical and typo errors
- Removed references to RDK

cs-abhishek-shukla committed:
1.0.0 changes
- Added usage section
- Rewrote Overview
- Removed other grammatical and typo errors
- Removed references to RDK
